### ISSUE TITLE
Update tests for du, ls, tail

### ISSUE THREAD

**Issue by shinhs0506** [uutils/coreutils#5010]
Some tests use [`expect result`](https://github.com/uutils/coreutils/blob/b03a88e154f8684f031f507811f7feb2b05600cf/tests/common/util.rs#L2458) which compares against local GNU coreutils installation which might be outdated. 

At the moment, Ubuntu-latest (22.04) in our CI/CD installs version 8.3x

**Comment by tertsdiepraam** [comment]
I agree, I think it's best if we rely on external tools as little as possible.

**Comment by n1000** [comment]
One difficulty that I ran into on OpenBSD is that any command that tries to display file size on disk (du, ls, etc) becomes hard to test across different OSes using approaches other than direct output comparison of the GNU coreutils.

Specifically on OpenBSD the filesystem blocksize and fragment size are "dynamic" (determined at install time based on the size of the install partition), and these different settings result in different expected output. I'm not sure if there is any easy to solution to this (although I am trying to learn more about what options there may be).

**Comment by RenjiSann** [comment]
I would classify the issue as wontfix, because sadly we don't really have the choice to refer to the installed reference when it comes to system-aware checks. One workaround would be to create a tool that gets and processes the information by itself, but that would pretty much mean implementing the very util we are testing.
